aws_iam_access_key.tf

```diff
@@ -5,7 +5,7 @@
 resource "aws_iam_access_key" "noAccessKeyForRootAccount" {
   user    = "root"
   pgp_key = "keybase:some_person_that_exists"
-  status  = "Active"
+  status  = "Inactive"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure no root account access key exists**
 
**Vulnerability**
The root account is the most privileged user in an AWS account. AWS Access Keys provide programmatic access to a given AWS account. It is recommended that all access keys associated with the root account be removed. Removing access keys associated with the root account limits vectors by which the account can be compromised. Additionally, removing the root access keys encourages the creation and use of role based accounts that are least privileged.
 
**Remediation**
Sign-in to the AWS Management Console, and select My Security Credentials in the navigation bar. Click Continue to Security Credentials. Expand the Access key section, and delete any keys that are listed there. Refer to: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_delete-key. For remediation, refer to control 1.12 in the CIS Amazon Web Services Foundations Benchmark v1.1.0 document that can be downloaded from: https://downloads.cisecurity.org/#/all 

source: accurics-